### PR TITLE
Fixed bug with annotated alias types.

### PR DESCRIPTION
**Pull request by boynton**
Fixes issue #56.

### CODE REVIEW

File: rdl/parser_test.go [hunk 1]
```diff
@@ -509,3 +509,22 @@ func TestAliasAnnotation(test *testing.T) {
...
+	   type MySubtype1 MyBase;
+	   type MySubtype2 MyBase (x_y);`)
+	if err != nil {
+		test.Errorf("cannot parse valid RDL: %v", err)
+		return
+	}
+	//
+	for _, td := range v.Types {
+		if td.AliasTypeDef != nil {
+			if td.AliasTypeDef.Type == "___forward_reference___" {
```
Comment (xythian) [inline rdl/parser_test.go:525]: Shouldn't this test also verify that the annotation survived?

**Review by xythian — DISMISSED** [review]
I think the test should verify that the annotation survived rather than just verifying that the internal state doesn't leak.

File: rdl/parser_test.go (hunk 1 above)
Comment (boynton) [reply rdl/parser_test.go:525]: Yes.

**Review by xythian — APPROVED** [review]
:+1: LGTM

**Pull request merged**